first_program.py

```diff
@@ -0,0 +1,6 @@
+"""First program"""
```

Comment:
D400 First line should end with a period
Just do not forget to put "." at the end of docstring -> """First program."""